<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeBased</class>
 <widget class="QWidget" name="AttributeBased">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>670</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>57</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Range :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset All</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>421</width>
     <height>527</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Area</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_15">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>Roundness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_14">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>Compactness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_13">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>Mean Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_12">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QRadioButton" name="radioButton_7">
        <property name="text">
         <string>Mean Green</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_9"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_10"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QRadioButton" name="radioButton_8">
        <property name="text">
         <string>Mean Blue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_11"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_12"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QRadioButton" name="radioButton_9">
        <property name="text">
         <string>Std Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_13"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_14"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QRadioButton" name="radioButton_10">
        <property name="text">
         <string>Std Green</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_15"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_16"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QRadioButton" name="radioButton_11">
        <property name="text">
         <string>Std Blue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_17"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_18"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QRadioButton" name="radioButton_12">
        <property name="text">
         <string>Perimeter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_19"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_20"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QRadioButton" name="radioButton_13">
        <property name="text">
         <string>Contrast</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_21"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_22"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QRadioButton" name="radioButton_14">
        <property name="text">
         <string>Roughness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_23"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_24"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QRadioButton" name="radioButton_15">
        <property name="text">
         <string>Coarseness</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLineEdit" name="lineEdit_25"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_26"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QRadioButton" name="radioButton_16">
        <property name="text">
         <string>Direction</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLineEdit" name="lineEdit_27"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_28"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QRadioButton" name="radioButton_17">
        <property name="text">
         <string>Class</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLineEdit" name="lineEdit_29"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_30"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_17">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Minimum Value :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_min">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Maximum Value :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_max">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_16</zorder>
  <zorder>textBrowser</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
